src/Calculator/Views/NumberPad.xaml.cpp

```diff
@@ -38,50 +38,50 @@ NumberPad::NumberPad() :
 
     const auto& localizationSettings = LocalizationSettings::GetInstance();
     
-    this->decimalSeparatorButton->Content = localizationSettings.GetDecimalSeparator();
-    this->num0Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('0');
-    this->num1Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('1');
-    this->num2Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('2');
-    this->num3Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('3');
-    this->num4Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('4');
+    this->DecimalSeparatorButton->Content = localizationSettings.GetDecimalSeparator();
+    this->Num0Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('0');
+    this->Num1Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('1');
+    this->Num2Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('2');
+    this->Num3Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('3');
+    this->Num4Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('4');
     this->num5Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('5');
-    this->num6Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('6');
-    this->num7Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('7');
-    this->num8Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('8');
-    this->num9Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('9');
+    this->Num6Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('6');
+    this->Num7Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('7');
+    this->Num8Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('8');
+    this->Num9Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('9');
 }
 
 void NumberPad::ProgModeRadixChange()
 {
-    num0Button->IsEnabled = true;
-    num1Button->IsEnabled = true;
-    num2Button->IsEnabled = true;
-    num3Button->IsEnabled = true;
-    num4Button->IsEnabled = true;
+    Num0Button->IsEnabled = true;
+    Num1Button->IsEnabled = true;
+    Num2Button->IsEnabled = true;
+    Num3Button->IsEnabled = true;
+    Num4Button->IsEnabled = true;
     num5Button->IsEnabled = true;
```

Comment:
What about `num5Button`?

Reply:
Thanks, I missed this because I left "match whole word" on.  After un-toggling it, I found and took care of a small handful of others (mostly control templates in App.xaml).

Here's confirmation of 100% completion in the UI project:

![image](https://user-images.githubusercontent.com/3520532/55042497-1b549800-5008-11e9-98f1-4dacbe4e5d3a.png)

Reply:
Here is the result of the tests after this commit:

![image](https://user-images.githubusercontent.com/3520532/55042768-763abf00-5009-11e9-833a-79388dacc315.png)